test/examples/src.latex_simple/index.md

```diff
@@ -28,3 +28,26 @@ julia> 127 % Int8
 ```julia-repl sayhello2
 julia> function foo end;
 ```
+
+## Issue 1401
+
+1. The output is a latex tabular environment
+
+```@example
+using DataFrames
+DataFrame(i=1:3, y='A':'C')
+```
+
+2. The paragraph itself is a latex tabular environment
+
+\begin{tabular}{r|ccc}
+        & i & y & z\\
+        \hline
+        & Int64 & Char & Int64\\
+        \hline
+        1 & 1 & 'A' & 5 \\
+        2 & 2 & 'B' & 6 \\
+        3 & 3 & 'C' & 7 \\
+        4 & 4 & 'D' & 8 \\
+\end{tabular}
```

Comment:
Coming back to https://github.com/JuliaDocs/Documenter.jl/pull/1403#discussion_r470538154: do you intend this to render as just text or do you want this to render as a table? Because it _should_ render as text.

Reply:
Yes, I intend to render as a table.

Reply:
Ok, but it _should_ render as text, because this is just text in a Markdown file. To include raw TeX like this, one should use an at-raw block:

````
```@raw latex
\begin{tabular}{r|ccc}
        & i & y & z\\
        \hline
        & Int64 & Char & Int64\\
        \hline
        1 & 1 & 'A' & 5 \\
        2 & 2 & 'B' & 6 \\
        3 & 3 & 'C' & 7 \\
        4 & 4 & 'D' & 8 \\
\end{tabular}
```
````

Reply:
Example 2 works as expected.
<img width="691" alt="Example-02" src="https://user-images.githubusercontent.com/10531698/90748648-07988880-e305-11ea-99d2-2af141a4d4ca.png">